bin/cron/form_geos

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env ruby
+#
+# This script incrementally populates the pegasus form_geos table using
+# geolocation of the existing ip_address.
+
+require File.expand_path('../../../pegasus/src/env', __FILE__)
+require 'cdo/db'
+require 'cdo/geocoder'
+
+BATCH_SIZE = 1000
+
+def main
+  time_now = DateTime.now
+  to_geolocate = PEGASUS_DB[:form_geos].where(indexed_at: nil).limit(1000)
+
+  to_geolocate.each do |form_geo|
+    location = Geocoder.search(form_geo[:ip_address]).first
+    PEGASUS_DB[:form_geos].where(id: form_geo[:id]).update(
+      updated_at: time_now,
+      indexed_at: time_now,
+      city: location.nil? || location.city.blank? ? nil : location.city,
```

Comment:
Nope, noting that `nil?` is applied to `location and `blank?` is applied to `location.city`.